core/Dockerfile

```diff
@@ -1,27 +1,30 @@
 
 FROM python:3.10.11-slim-bullseye
 
-### PREPARE BUILD WITH NECESSARY FILES AND FOLDERS ###
-RUN mkdir -p /app && mkdir -p /admin
-COPY ./pyproject.toml /app/pyproject.toml
-COPY ./cat /app/cat
-COPY ./install_plugin_dependencies.py /app/install_plugin_dependencies.py
-
 ### SYSTEM SETUP ###
 RUN apt-get -y update && apt-get install -y curl build-essential fastjar libmagic-mgc libmagic1 mime-support && \
     apt-get clean && \
     rm -rf /var/lib/apt/lists/*
 
 ### ADMIN (static build) ###
 WORKDIR /admin
-RUN curl -sL https://github.com/cheshire-cat-ai/admin-vue/releases/download/Admin/release.zip | jar -xv
+RUN curl -sL https://github.com/cheshire-cat-ai/admin-vue/releases/download/Admin/develop.zip | jar -xv
+
+### PREPARE BUILD WITH NECESSARY FILES AND FOLDERS ###
+COPY ./pyproject.toml /app/pyproject.toml
 
-### INSTALL PYTHON DEPENDENCIES (Core and Plugins) ###
+### INSTALL PYTHON DEPENDENCIES (Core) ###
 WORKDIR /app
 RUN pip install -U pip && \
     pip install --no-cache-dir . &&\
-    python3 -c "import nltk; nltk.download('punkt');nltk.download('averaged_perceptron_tagger')" &&\
-    python3 install_plugin_dependencies.py
+    python3 -c "import nltk; nltk.download('punkt');nltk.download('averaged_perceptron_tagger')"
+
+### COPY PLUGINS ###
+COPY ./cat/plugins /app/cat/plugins
+
+### INSTALL PYTHON DEPENDENCIES (Plugins) ###
+COPY ./install_plugin_dependencies.py /app/install_plugin_dependencies.py
+RUN python3 install_plugin_dependencies.py
 
 ### FINISH ###
-CMD python3 -m cat.main
```

Comment:
We need this CMD in case you launch the image solo (out of docker compose)

Reply:
Please the PR branched from `develop` and towards `develop`
(Because there are commits on `main` that are not on `develop` and vice versa)